packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -811,7 +819,7 @@ class DateTimePicker extends React.Component {
             value={dateValue}
             name="input"
             disabled={disabled}
-            disableButtonFocusOnClose
+            disableButtonFocusOnClose={disableButtonFocusOnClose}
```

Comment:
Instead of setting it to false you can remove this prop completely. But I see that it was added to fix a focus [issue](https://github.com/cerner/terra-framework/pull/699#pullrequestreview-247298099) on firefox. Can you test it once?

Reply:
Removed the prop completely. In firefox these changes are working fine.